react/src/components/atoms/media/SiteLogo/SiteLogo.stories.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import { storiesOf } from '@storybook/react';
 import { withKnobs, text, number } from '@storybook/addon-knobs';
-import logo from 'SharedAssets/images/stateseal.png';
+import logo from '../../../../assets/images/stateseal.png';
```

Comment:
Can we add back the `sharedAssets` alias? We might want to change the path in the future if we can find a better solution than using creating symlinked folder from `/assets`